tf/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "aws_region" {
+  type        = string
+  description = "aws region"
+  default     = "eu-central-1"
+}
+
+variable "log_level" {
+  type        = string
+  description = "lambda extension log level"
+  default     = "trace"
+}
+
+variable "ess_region" {
+  type        = string
+  description = "ess region"
+  default     = "aws-eu-central-1"
```

Comment:
Should we default `ess_region` to `gcp-us-west2` (CFT/Cloud First Testing region), since it's where we do the rest of our cloud-based testing?

Although it's crossing regions, the amount of traffic will be minimal - and running the deployment in the dedicated CFT region means we won't be contending with customers for ESS resources.

Reply:
Agree! Moved 👍